aemedge/blocks/image-slider/image-slider.css

```diff
@@ -2,7 +2,7 @@
 .carousel-slides {
   display: flex;
   gap: 8px;
-  padding: 16px;
+  padding: unset;
```

Comment:
Actually can you leave the 16px? Although it is not in their original, I remember we talked about adding this so the end user would have a clue that there were more images to see...